cdap-ui/app/cdap/components/AbstractWidget/FormInputs/MultiSelect/index.tsx

```diff
@@ -42,8 +42,8 @@ export default function MultiSelect({ value, delimiter, options, onChange }: IMu
     onChange(selectionsString);
   };
 
-  const optionValues = options.map((opt: IOption) => {
-    return typeof opt === 'string' ? opt : opt.label;
+  const optionValues = options.map((opt) => {
```

Comment:
nit/for future: options.map(opt => opt.label);

Reply:
Changed, with a small variation since the linter prefers (opt) => opt.label